service/src/main/java/skills/controller/AdminController.groovy

```diff
@@ -926,12 +927,13 @@ class AdminController {
                                 @RequestParam int limit,
                                 @RequestParam int page,
                                 @RequestParam String orderBy,
-                                @RequestParam Boolean ascending) {
+                                @RequestParam Boolean ascending,
+                                @RequestParam int minimumPoints) {
         SkillsValidator.isNotBlank(projectId, "Project Id")
         SkillsValidator.isNotBlank(subjectId, "Subject Id", projectId)
 
```

Comment:
Validate that `minimumPoints` is valid, probably using `SkillsValidator`; add service test(s) to verify `minimumPoints` validation